src/sql/base/common/locConstants.ts

```diff
@@ -38,6 +38,8 @@ export function extensionsWorkbenchServiceIncompatible(extension: string, extens
 export const fileActionsContributionNewQuery = localize('newQuery', "New Query");
 export const fileActionsContributionMiNewQuery = localize({ key: 'miNewQuery', comment: ['&& denotes a mnemonic'] }, "New &&Query");
 export const fileActionsContributionMiNewNotebook = localize({ key: 'miNewNotebook', comment: ['&& denotes a mnemonic'] }, "&&New Notebook");
+export const fileActionsContributionNewDeployment = localize('newDeployment', "New Deployment...");
+export const fileActionsContributionMiNewDeployment = localize({ key: 'miNewDeployment', comment: ['&& denotes a mnemonic'] }, "&&New Deployment...");
```

Comment:
Why do some values denote a mnemonic? Added both to be consistent.

Reply:
Because some are used in places where mnemonics aren't valid - like the command palette. Search for users of `fileActionsContributionNewQuery` and add the same there

(New Notebook is slightly special it seems in that it's contributed partly by the notebook extension so does its command palette entries through that)